README.md

```diff
@@ -44,21 +44,23 @@ npm install --save agendash
 Agendash comes with a standalone Express app which you can use like this:
 
 ```bash
-./node_modules/.bin/agendash --db=mongodb://localhost/agendaDb --collection=agendaCollection --port=3001
+./node_modules/.bin/agendash --db="mongodb://localhost/agendaDb" --collection=agendaCollection --port=3001
 ```
 
 or like this, for default collection `agendaJobs` and default port `3000`:
 
 ```bash
-./node_modules/.bin/agendash --db=mongodb://localhost/agendaDb
+./node_modules/.bin/agendash --db="mongodb://localhost/agendaDb"
 ```
 
 If you are using npm >= 5.2, then you can use [npx](https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b):
 
 ```bash
-npx agendash --db=mongodb://localhost/agendaDb --collection=agendaCollection --port=3001
+npx agendash --db="mongodb://localhost/agendaDb" --collection=agendaCollection --port=3001
 ```
 
+Then browse to `http://localhost:3000/`
```

Comment:
Should be `3001` as used above.

Reply:
@OmgImAlexis  I see your point but don't' totally agree because the default port is 3000 so I think if there is one link it should be the default port.  What I did was update the readme to add a link for each of the standalone examples.  Does that seem reasonable?